public/v2/apps/collabora.json

```diff
@@ -0,0 +1,57 @@
+{
+    "captainVersion": "2",
+    "documentation": "Inspired by https://www.collaboraoffice.com/code/docker/",
+    "displayName": "Collabora Online",
+    "description": "Collabora Online Developpement Edition is an online and collaborating office suite",
+    "dockerCompose": {
+        "version": "1",
+        "services": {
+            "$$cap_appname": {
+                "image": "collabora/code:$$cap_collabora_version",
+                "containerHttpPort": "9980",
+                "restart": "always",
+                "environment": {
+                  "domain": "$$cap_wopihost_domain",
+                  "cert_domain": "$$cap_appname.$$cap_root_domain",
+                  "server_name": "$$cap_appname.$$cap_root_domain",
+                  "extra_params": "--o:ssl.enable=false --o:ssl.termination=true",
+                  "username": "$$cap_admin_user",
+                  "password": "$$cap_admin_pass"
+                }
+            }
+        }
+    },
+    "instructions": {
+        "start": "Collabora CODE version is an online Office Suite. \nYou need a runngin instance of Nextcloud or  ownCloud\nThe process will take about a minute for the process to finish.",
```

Comment:
Also, let's add a link to their Docker page as it contains more info: https://www.collaboraoffice.com/code/docker